torch/nn/quantized/modules/functional_modules.py

```diff
@@ -71,6 +72,13 @@ def cat(self, x, dim=0):
         r = self.activation_post_process(r)
         return r
 
+    r"""Operation equivalent to ``torch.clamp(Tensor, float, float)``"""
+    def clamp(self, x, min_, max_):
+        # type: (Tensor, float, float) -> Tensor
+        r = torch.clamp(x, min_, max_)
+        r = self.activation_post_process(r)
+        return r
+
```

Comment:
we probably don't need this if clamp is using scale/zero_point from input

Reply:
Let me remove this.